### ISSUE TITLE
Problem with Deployment

### ISSUE THREAD

**Issue by SteveWedge** [OfficeDev/microsoft-teams-apps-company-communicator#1560]
Not sure what the issue is here and would appreciate some help

![image](https://github.com/user-attachments/assets/8f9df1bf-dbf1-4874-a100-0da6a348f10f)

**Comment by peddivyshnavi** [comment]
Hi @SteveWedge 
We recommend you follow the manual deployment over PowerShell approach as there is existing issue script that is not compatible with Azure CLI version. In addition, good troubleshooting is possible with manual approach.
[Manual approach](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/wiki/Deployment-guide)

**Comment by SteveWedge** [comment]
OK, thanks, so it would seem that the recommended option in your documentation is, in fact, not recommended. Maybe that needs to be amended?

![image](https://github.com/user-attachments/assets/ae246932-50ee-4d75-a23b-fe557b6df045)

**Comment by peddivyshnavi** [comment]
Hi @SteveWedge 
As mentioned earlier PowerShell approach there is existing issue script that is not compatible with Azure CLI version in future sprints we will resolve this issue

**Comment by peddivyshnavi** [comment]
@SteveWedge 
Could you please update us on the status of the issue?

**Comment by SteveWedge** [comment]
I have managed to get back to this today. 
I did have an issue with setting up a custom domain, but that was more to do with trying to deploy a non Microsoft generated certificate for the Binding from a Key Vault. I eventually gave up and just used the Microsoft provided cert.

The other thing to note is that the creation of the Manifest.json file for the Users App does require <<appDomain>> to be supplied and that isn't mentioned in the final paragraph, at the end of Step 5.

Both packages are now deployed and I'm currently waiting for them to be available to be added into Teams so that we can trial this.
